UnityProject/Assets/Scripts/Objects/Botany/SeedExtractor.cs

```diff
@@ -147,6 +148,11 @@ public void ServerPerformInteraction(HandApply interaction)
 			networkTab.ServerPerformInteraction(interaction);
 		}
 
+		public bool WillInteract(HandApply interaction, NetworkSide side)
+		{
+			return DefaultWillInteract.Default(interaction, side);
```

Comment:
```suggestion
    return DefaultWillInteract.Default(interaction, side) &&
               interaction.TargetObject == gameObject &&
               interaction.HandObject != null &&
               interaction.HandObject.TryGetComponent<GrownFood>(out _);
```

don't directly apply the suggestion, I had to use spaces and some names might not be right.